### PR TITLE
Make sending events to Jembi optional

### PR DESCRIPTION
**Pull request by rudigiesler**
This has 2 parts.

First, we need to have a flag that we can set to stop sending events.

Second, we need a command we can run to resend any events that weren't sent

### CODE REVIEW

File: registrations/tests.py [hunk 1]
```diff
@@ -3869,6 +3723,36 @@ def test_send_outgoing_message_to_jembi(self):
...
+            "inbound_channel_id": "6a5c691e-140c-48b0-9f39-a53d4951d7fa",
+            "message_id": message_id,
+        }
+        # Execute
+        response = self.normalclient.post(
+            "/api/v1/jembi/helpdesk/outgoing/", user_request
+        )
+
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(len(responses.calls), 1)
```
Comment (KaitCrawford) [inline registrations/tests.py:3754]: What is the call being made here?

Comment (rudigiesler) [reply registrations/tests.py:3754]: It's the call to Junebug, so probably not the best test to pick, but you can see the previous test asserts 2 calls, and this one asserts one

Comment (KaitCrawford) [reply registrations/tests.py:3754]: ok

**Review by KaitCrawford — APPROVED** [review]
:+1:

**Pull request merged**